configure.ac

```diff
@@ -4003,11 +4003,18 @@ AC_ARG_ENABLE(rjit,
 )
 
 AS_CASE(["$RJIT_SUPPORT"],
-[yes|dev], [
+[yes|dev|disasm], [
     AS_CASE(["$RJIT_SUPPORT"],
     [dev], [
         # Link libcapstone for --rjit-dump-disasm
         AC_CHECK_LIB([capstone], [cs_disasm])
+
+        # Enable extra stats (vm_insns_count, ratio_in_rjit)
+        AC_DEFINE(RJIT_STATS, 1)
+    ],
+    [disasm], [
+        # Link libcapstone for --rjit-dump-disasm
+        AC_CHECK_LIB([capstone], [cs_disasm])
```

Comment:
I realize this was part of the revert but it looks a little funny...
To confirm, this is adding stats to the dev build,
And adding a disasm build in case anyone wants disasm without stats
(but there is no build for stats without disasm).